### DISCUSSION TITLE
Switch time controlled

### DISCUSSION THREAD

**Discussion by cmat21** [ra3xdh/qucs_s#348] [Ideas]
Hi, is there any possibility to add a "switch time controlled" component to ngspice?
![Switch_time_controlled](https://github.com/ra3xdh/qucs_s/assets/70080894/06dd839f-da68-4e72-ac49-b44321f039f9)

**Comment by ra3xdh** [comment]
I will look this. In principle such device could be added.

**Comment by ra3xdh** [comment]
Switch implementation added by #355 This device will be available since the next release.
